doc/source/tutorial/stats/discrete_nhypergeom.rst

```diff
@@ -0,0 +1,23 @@
+
+.. _discrete-nhypergeom:
```

Comment:
Need to add to https://github.com/scipy/scipy/blob/master/doc/source/tutorial/stats/discrete.rst

Reply:
Did it! Thanks!